ui/portal_work.js

```diff
@@ -273,9 +299,8 @@ class PortalWork extends ClassUI {
                   sideBar.select(foundMenuEntry.id);
                   this.selectApplication(foundMenuEntry.id);
 
-                  var defaultPageID = this.AppState.lastPages[
-                     foundMenuEntry.abApplication.id
-                  ];
+                  var defaultPageID =
+                     this.AppState.lastPages[foundMenuEntry.abApplication.id];
```

Comment:
Are these changes what eslint/prettier are doing?  I wonder if there is a settings difference, or if you have the latest versions installed ... which you should since you just performed an install ...  (maybe my version is behind yours?)